app/models/bot/tagger.rb

```diff
@@ -0,0 +1,65 @@
+require 'json'
+
+class Bot::Tagger < BotUser
+  check_settings
+  class Error < ::StandardError
+  end
+
+  def self.get_tag_text(tag_id,auto_tag_prefix,ignore_autotags)
+    tag=TagText.find_by_id(tag_id).text
+    if tag.nil? || (ignore_autotags && tag[0]==auto_tag_prefix)
+      tag=nil
```

Comment:
Ah, it's because in Ruby the last line is like the `return`. So `tag = nil` could be simply rewritten as `nil`.

Reply:
ah, that is true. I didn't think of it that way. I was thinking it was complaining about 
```if tag.nil?
      tag = nil
   ...
```
as being useless, which it would be if the if statement didn't have the second part after the boolean or. I'll update. 

Codeclimate wins this time, but watch out :eye:  :joy: